packages/runner/src/scripts/build.ts

```diff
@@ -0,0 +1,14 @@
+import { execSync } from "child_process";
```

Comment:
Had to add this file and the one below it into `src`, even though they're just scripts.
Could change the `tsconfig.json`, but perhaps we want to think about some sort of convention?

These aren't meant to be source files, but scripts to be run during development, etc.